src/mkdocs_bibtex/registry.py

```diff
@@ -146,28 +146,29 @@ def validate_citation_blocks(self, citation_blocks: list[CitationBlock]) -> None
                 if not all(citation.key in self._reference_cache for citation in block.citations)
             ]
 
-        if len(unprocessed_blocks) > 0:
+        if unprocessed_blocks:
```

Comment:
This is technically valid python, but generally not a good idea. I prefer the length check.
```suggestion
        if len(unprocessed_blocks) > 0:
```